### ISSUE TITLE
UndefinedBehaviorError uncaught; freeze SBT

### ISSUE THREAD

**Issue by japgolly** [lihaoyi/utest#89]
Errors like this in tests:

```
scala.scalajs.runtime.UndefinedBehaviorError: An undefined behavior was detected: undefined is not an instance of java.lang.Integer
```

aren't caught by μtest, and so they freeze SBT without any visual output.

It's probably because they're defined as fatal errors. You might be using `Try` or have a `case NonFatal(t) =>` somewhere. It should change to just a `try ??? catch {case  t: Throwable => ??? }`.

**Comment by tekacs** [comment]
Yeah it's this:
https://github.com/lihaoyi/utest/blob/927cebc46985e06f4f48e81de2c08097a1b4b1ad/utest/shared/src/main/scala/utest/asserts/package.scala#L51

Used here:
https://github.com/lihaoyi/utest/blob/927cebc46985e06f4f48e81de2c08097a1b4b1ad/utest/shared/src/main/scala/utest/asserts/Asserts.scala#L128

**Comment by japgolly** [comment]
Resolved in 0.4.5 via #92
